auth/api/http/groups/endpoint.go

```diff
@@ -358,11 +358,15 @@ func buildUsersResponse(mp auth.MemberPage) memberPageRes {
 			Limit:  mp.Limit,
 			Name:   mp.Name,
 		},
-		Members: []interface{}{},
+		Members: []memberRes{},
 	}
 
-	for _, m := range mp.Members {
-		res.Members = append(res.Members, m)
+	for _, member := range mp.Members {
+		memb := memberRes{
+			ID:   member.ID,
+			Type: member.Type,
```

Comment:
Do we really need type here? It looks like it should be a part of `memberPageRes` because all the members are of the same type.

Reply:
done

Reply:
Do we even need `memberRes` since we only use ID? Does it make sense to return only a list of strings that are IDs?